sections/featured-blog.liquid

```diff
@@ -9,7 +9,106 @@
 
   {% unless blog.articles_count == 0 %}
     <ul class="grid grid--uniform grid--blog">
+      
+<!--       Loop through and check if there is a featured article -->
+     {% for article in blog.articles limit: section.settings.post_limit %}
```

Comment:
The problem with this code is that it only loops through a specific number of articles not all of them.